<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QDialog" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>1. What is COVID-19?
    1) COVID-19 is a disease caused by a virus called SARS-CoV-2.
    2) COVID-19 is a storm's name in the U.S.
    3) COVID-19 is some kind of a cancer disease.
    4) None of the above.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 171, 228);</string>
   </property>
   <property name="text">
    <string>submit</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="userAnswer1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Result1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>681</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>2. How does COVID-19 is spreading?
    1) Touching eyes, nose, or mouth with hands that have the virus on them.
    2) Breathing in air when close to an infected person.
    3) Small droplets and particles that contain virus land on the eyes, nose, or mouth.
    4) All of the above.</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="userAnswer2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 171, 228);</string>
   </property>
   <property name="text">
    <string>submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="Result2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>3. If you have symptoms of COVID-19, what should you need to do?
    1) Stay at home (except to get tested or to get medical care).
    2) Go outside to hangout with your friends.
    3) Meet only with those who are also infected.
    4) Do not eat anything until your are healed.</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="userAnswer3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Result3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>400</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 171, 228);</string>
   </property>
   <property name="text">
    <string>submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>601</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>4. If you receive a positive test result, what you should do?
    1) Go to test again, maybe it is a mistake.
    2) Go to your parents house for them to treat you.
    3) Take an Advil.
    4) Stay home for at least 5 days and isolate from others in your home.</string>
   </property>
  </widget>
  <widget class="QLabel" name="Result4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="userAnswer4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>550</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 171, 228);</string>
   </property>
   <property name="text">
    <string>submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>5. When should you seek emergency care if I have COVID-19?
    1) Trouble breathing, persistent pain or pressure in the chest.
    2) If you start to laugh a lot.
    3) If you feel depressed .
    4) You have slightly fever.</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="userAnswer5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>700</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Result5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>740</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>700</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 171, 228);</string>
   </property>
   <property name="text">
    <string>submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>6. Where was COVID-19 first identified?
    1) United States.
    2) India.
    3) Germany.
    4) China.</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="userAnswer6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Result6">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button6">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>100</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 171, 228);</string>
   </property>
   <property name="text">
    <string>submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label7">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>150</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>7. When was COVID-19 first identified?
    1) 2022.
    2) 2018.
    3) 2011.
    4) 2019.</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="userAnswer7">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>250</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Result7">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>280</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button7">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>250</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 171, 228);</string>
   </property>
   <property name="text">
    <string>submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label8">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>300</y>
     <width>361</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>8. What is Omicron?
    1) A type of vegetable.
    2) A new movie in the movie theaters.
    3) the last variant known of COVID-19.
    4) a rare animal which discovered lately in Madagascar.</string>
   </property>
  </widget>
  <widget class="QLabel" name="Result8">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="userAnswer8">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>390</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button8">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>390</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 171, 228);</string>
   </property>
   <property name="text">
    <string>submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label9">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>440</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>9. Are the COVID-19 vaccines safe and effective?
    1) No, you should watch yourself and avoid it.
    2) It is unknown .
    3) Yes.
    4) Only on adults.</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="userAnswer9">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>530</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Result9">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>560</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button9">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>530</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 171, 228);
</string>
   </property>
   <property name="text">
    <string>submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label10">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>580</y>
     <width>451</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>10. You have been vaxxed, do you still need to get tested for COVID-19?
    1) Yes, If you have symptoms of the virus, contact your doctor.
    2) You don't need if you are fully vaccinated, even if you have symptoms.
    3) Only if the symptoms are getting worse.
    4) None of the above.</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="userAnswer10">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>670</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button10">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>670</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(152, 171, 228);</string>
   </property>
   <property name="text">
    <string>submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="Result10">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>700</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>740</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 3, 0);</string>
   </property>
   <property name="text">
    <string>Exit Game</string>
   </property>
  </widget>
  <widget class="QLabel" name="score">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>740</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(1, 18, 175)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="timer">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="score2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>730</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(1, 18, 175)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="links">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>740</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-image: url(:/img/7419b80742274fc72231166504b06394.jpeg);</string>
   </property>
   <property name="text">
    <string>Links</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelScore">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>710</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelScore2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>710</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="game_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-100</y>
     <width>1081</width>
     <height>1021</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1081</width>
     <height>1021</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>game_background</zorder>
  <zorder>label1</zorder>
  <zorder>button1</zorder>
  <zorder>userAnswer1</zorder>
  <zorder>Result1</zorder>
  <zorder>label2</zorder>
  <zorder>userAnswer2</zorder>
  <zorder>button2</zorder>
  <zorder>Result2</zorder>
  <zorder>label3</zorder>
  <zorder>userAnswer3</zorder>
  <zorder>Result3</zorder>
  <zorder>button3</zorder>
  <zorder>label4</zorder>
  <zorder>Result4</zorder>
  <zorder>userAnswer4</zorder>
  <zorder>button4</zorder>
  <zorder>label5</zorder>
  <zorder>userAnswer5</zorder>
  <zorder>Result5</zorder>
  <zorder>button5</zorder>
  <zorder>label6</zorder>
  <zorder>userAnswer6</zorder>
  <zorder>Result6</zorder>
  <zorder>button6</zorder>
  <zorder>label7</zorder>
  <zorder>userAnswer7</zorder>
  <zorder>Result7</zorder>
  <zorder>button7</zorder>
  <zorder>label8</zorder>
  <zorder>Result8</zorder>
  <zorder>userAnswer8</zorder>
  <zorder>button8</zorder>
  <zorder>label9</zorder>
  <zorder>userAnswer9</zorder>
  <zorder>Result9</zorder>
  <zorder>button9</zorder>
  <zorder>label10</zorder>
  <zorder>userAnswer10</zorder>
  <zorder>button10</zorder>
  <zorder>Result10</zorder>
  <zorder>pushButton</zorder>
  <zorder>score</zorder>
  <zorder>timer</zorder>
  <zorder>score2</zorder>
  <zorder>links</zorder>
  <zorder>labelScore</zorder>
  <zorder>labelScore2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
